task1/src/test/java/com/fundacionjala/movies/MovieTest.java

```diff
@@ -0,0 +1,16 @@
+package com.fundacionjala.movies;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import static org.junit.Assert.assertNotNull;
+
+public class MovieTest {
```

Comment:
This unit test class is not necessary because this class is abstract
Remove it.